staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -380,7 +380,10 @@ func RecordRequestAbort(req *http.Request, requestInfo *request.RequestInfo) {
 	}
 
 	scope := CleanScope(requestInfo)
-	reportedVerb := cleanVerb(CanonicalVerb(strings.ToUpper(req.Method), scope), "", req)
+	reportedVerb := getVerbIfWatch(req)
+	if reportedVerb != "WATCH" {
+		reportedVerb = cleanVerb(CanonicalVerb(strings.ToUpper(req.Method), scope), "", req)
```

Comment:
Let's unify the calls across all calls and change to:
```
reportedVerb := cleanVerb(CanonicalVerb(strings.ToUpper(req.Method), scope), getVerbIfWatch(req), req)
```

[Same everywhere below]